### ISSUE TITLE
Tensorflow Imlementation

### ISSUE THREAD

**Issue by adi1999** [JLrumberger/SpatialEmbeddingLoss#1]
Thanks for the tensorflow implementation of the loss function, were you able to implement the paper too in tensorflow?

**Comment by JLrumberger** [comment]
Sure, I used the U-Net implementation from [here ](https://github.com/funkelab/funlib.learn.tensorflow) used imgaug for augmentations and trained on the BBBC010 dataset from the broad institute. The lovasz loss implementation is super slow because of sort() and I replaced it for other experiments with a standard binary cross-entropy loss.

**Comment by adi1999** [comment]
Great, I would be great help if you're able to provide you tensorflow implementations.
Also, I had one more doubt, for the function "joint_loss_single" you've used "embeddings, sigma, correct_label" as input instead of 
" prediction, instances, labels" which are used in neven's function?

**Comment by adi1999** [comment]
Hey @JLrumberger , thanks for your response, Can you also help me to understand that how can we obtain offset vectors and sigma from [here](https://github.com/davyneven/SpatialEmbeddings/blob/master/src/train.py)   which you've used as a input to your loss function.
